Add one more reporting api service to COOP/COEP article (#4910)
GoogleChrome/web.dev: src/site/content/en/blog/coop-coep/index.md

##### Use a solution that accepts reports

There are a couple of solutions that accept reports from the browser's COOP/COEP
reporting functionality:

* [https://report-uri.com](https://report-uri.com)
* [https://uriports.com](https://uriports.com)

> If there's any other solutions that accept reports, [please let us know to

Reviewer: ```suggestion
If there are any other solutions that accept reports, [let us know to
```

> We are aware of one solution that accepts reports from the browser's COOP/COEP
> reporting functionality.

Reviewer: ```suggestion
There are a couple of solutions that accept reports from the browser's COOP/COEP
reporting functionality:
```
